src/Nbody/Sinks.cpp

```diff
@@ -463,6 +461,29 @@ void Sinks<ndim>::AccreteMassToSinks
     //---------------------------------------------------------------------------------------------
 
 
+    // Determine the closest sinks to all particles that are inside sinks
+#pragma omp for schedule(dynamic,1)
```

Comment:
No no, this loop is over Nhydro, this is the problem. Loops of order N_particles should be avoided!!!!!